NickvisionCavalier.GNOME/Views/DrawingView.cs

```diff
@@ -37,13 +38,23 @@ public partial class DrawingView : Gtk.Stack, IDisposable
     private SKSurface? _skSurface;
     private float[]? _sample;
 
-    private DrawingView(Gtk.Builder builder, DrawingViewController controller) : base(builder.GetPointer("_root"), false)
+    private DrawingView(Gtk.Builder builder, MainWindow window, DrawingViewController controller) : base(builder.GetPointer("_root"), false)
```

Comment:
I'd simplify `MainWindow window` to just `Gtk.Window parent` since we don't need access to `MainWindow` here just the underlying `Gtk.Window`.